src/cov.jl

```diff
@@ -44,80 +37,73 @@ that the data are centered and hence there's no need to subtract the mean.
 When `vardim = 1`, the variables are considered columns with observations in rows;
 when `vardim = 2`, variables are in rows with observations in columns.
 """
-function scattermat end
```

Comment:
Ah, okay, to clarify, the preference moving forward is that we want the docstring attached to `function <name> end` with all other methods below that?